### ISSUE TITLE
Use TDD consistently

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-discovery#410]
"Directory" is a generic term, we should use TDD consistently to refer to the Thing Description Directory service.

**Comment by farshidtz** [comment]
Is there a case where we need to refer to directories in general within the specification? TDD is too wordy.

**Comment by mmccool** [comment]
still needs to be done, but editorial, can do during REC

**Comment by mmccool** [comment]
In general, use of "directory" (lower case) is probably ok, and not worth re-opening the REC to change at this point.  Let's revisit this terminology however for the next version.  Will change the label so it's still open, but for Discovery 1.1.
